public/locales/en/common.json

```diff
@@ -9,5 +9,6 @@
     "moreInformation": "More Information",
     "jumpTo": "Jump to",
     "aqiHelpHeading": "What Is AQI?",
-    "aqiHelpMessage": "AQI stands for Air Quality Index. It is the national standard for indicating how good or bad the air is. Lower numbers indicate safer air, while higher numbers indicate more hazardous air."
+    "aqiHelpMessage": "AQI stands for Air Quality Index. It is the national standard for indicating how good or bad the air is. Lower numbers indicate safer air, while higher numbers indicate more hazardous air.",
+    "noSensorGauge": "Click a sensor on the map to see its AQI value here!"
```

Comment:
Since `noSensorGauge` is no longer the case here, maybe let's call is `instructions` or something like that